go.mod

```diff
@@ -322,6 +319,8 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 )
 
+require github.com/libp2p/go-libp2p-pubsub v0.4.2 // direct
```

Comment:
kinda weird that this got moved out of the first block :) I also don't get this with 1.17.5. perhaps undo it manually? it might be a leftover from the earlier mess with 1.16.